content/ngf/overview/gateway-api-compatibility.md

```diff
@@ -389,3 +387,5 @@ Custom policies are NGINX Gateway Fabric-specific CRDs (Custom Resource Definiti
 While these CRDs are not part of the Gateway API, the mechanism to attach them to Gateway API resources is part of the Gateway API. See the [Policy Attachment documentation](https://gateway-api.sigs.k8s.io/references/policy-attachment/).
 
 See the [custom policies]({{< ref "/ngf/overview/custom-policies.md" >}}) document for more information.
+
+---
```

Comment:
Is this necessary at the end?

Reply:
@ADubhlaoich is your thumbs meaning it should exist or should not exist?

Reply:
My bad, didn't realise how ambiguous that was.

It shouldn't exist, we're gradually excising horizontal lines from markdown.

As a stylistic element, the newer docs theme instead has better whitespace to meaningfully delineate sectiions.